src/data/spark-miner-rsr-summaries.json.js

```diff
@@ -0,0 +1,13 @@
+import { jsonFetcher } from "./json-fetcher.js";
+import { getDateXDaysAgo } from "../utils/date-utils.js";
+
+const summaries = {};
+
+for (let i = 201; i >= 1; i--) {
+  const dayString = getDateXDaysAgo(i);
+  summaries[dayString] = await jsonFetcher(
+    `https://stats.filspark.com/miners/retrieval-success-rate/summary?from=${dayString}&to=${dayString}`
+  );
+}
+
+process.stdout.write(JSON.stringify(summaries));
```

Comment:
Forgotten log?

Reply:
Nope, without this this data source wouldn't produce any data

Reply:
These scripts are expected to write their data to stdout

Reply:
Oh, interesting. OK!